TACC/Core-Portal
# FP-649: Make disabled links be gray

## Status: ##

Closed until ticket is in progress, at which point branch/PR could be updated/re-opened.

## To Do: ##

- [ ] Test thoroughly across app.
    - [ ] Upload screenshots.

## Overview: ##

Make disabled links be gray.

## PR Status: ##


## Related Jira tickets: ##

* [FP-649](https://jira.tacc.utexas.edu/browse/FP-649)

## Summary of Changes: ##

- __Major__: Globally, make disabled `.btn-link`'s gray.
- __Minor__: Duplicate stlyes in the two existing places that have `.btn-link` styles (comments exist to explain why).

## Testing Steps: ##
1. Visit these locations and check that the listed link-buttons, when disabled, have light gray (dark gray but low opacity) text:
    - Applications: Reset Fields to Defaults"
    - …

## UI Photos:

### Sample of Change

- Enabled:\
    <img width="544" alt="Enabled" src="https://user-images.githubusercontent.com/62723358/90798480-99899c80-e2d7-11ea-945b-f99c6af0665a.png">

- Disabled (Before):\
    <img width="544" alt="Disabled - Purple, Opactiy Drop" src="https://user-images.githubusercontent.com/62723358/90798477-98f10600-e2d7-11ea-9213-b5c52394cbd4.png">

- Disabled (After):\
    <img width="544" alt="Disabled - Gray,  Opacity Drop" src="https://user-images.githubusercontent.com/62723358/90798484-9abac980-e2d7-11ea-931f-af674707843e.png">

#### Proof of Consistent Global Change

(pending testing)
